card.go

```diff
@@ -219,10 +219,6 @@ func (card *Card) getExtraMetadata(inputURL string) {
 				cm.PreviousFlavourTexts = append(cm.PreviousFlavourTexts, c.FlavourText)
 			}
 			cm.PreviousPrintings = append(cm.PreviousPrintings, c.formatExpansions())
-			// Only need previous reminder text if current one doesn't have
-			if c.getReminderTexts() == "Reminder text not found" && c.getReminderTexts() != "Reminder text not found" {

```

Comment:
Don't delete
The correct check is:
`if card.getReminderTexts() == "Reminder text not found" && c.getReminderTexts() != "Reminder text not found" {`